study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/study/addOrEditStudyNotification.jsp

```diff
@@ -142,7 +142,9 @@
           <div class="help-block with-errors red-txt"></div>
         </div>
       </div>
-
+ <div class="form-group mr-sm" style="white-space: normal; margin-top: -9px;">
+ For studies that are live, notifications get published for scheduled delivery to participants as soon as you mark the Notifications section complete.
```

Comment:
This is an awkward sentence - please ask tech writer for help

Reply:
Fixed